src/styles/components/_component-grid.scss

```diff
@@ -126,10 +134,97 @@ $breakpoint-tablet: 820px;
     &--max-scott {
       grid-area: max-scott;
       background-color: transparent;
-      width: 77%; // this closely matches the dimensions in Figma
+      width: 65%; // this closely matches the dimensions in Figma
```

Comment:
Changing this is a little out of scope, but but at 77% this image of Max didn't leave much room for squiggles. It also seemed to be a bit larger than the design, but I wasn't sure how to check with the percentage width.

Reply:
I'd probably take a different tactic and remove the `width` property here. Then I'd make the image itself have a max-width and auto height, looks like 275px wide in Figma.